kevinzs/day02/day02.py

```diff
@@ -0,0 +1,20 @@
+def solver(input):
+    res = 0
+    for row in input:
+        min_value = min(int(x) for x in row)
+        max_value = max(int(x) for x in row)
+        res += max_value - min_value
+    return res
```

Comment:
This solution is really easy 👍 

Did you know that you can make a mapreduce solution in Python too?